File: Easy/Body Weight is a Girls Secret/Body Weight Is a Girls Secret.cs
using System;
using System.Linq;

class Solution
{
    static void Main(string[] args)
    {
        int[] inputs = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();
        int c = (inputs.Sum()/4)-inputs[0]-inputs[9];
        int a = inputs[1]-c;
        int b = inputs[0]-(inputs[1]-c);
        int e = inputs[8]-c;
        int d = inputs[9]-(inputs[8]-c);

        Console.WriteLine($"{a} {b} {c} {d} {e}");
    }
}

File: Easy/Body Weight is a Girls Secret/Body Weight Is a Girls Secret.py
weights = []
answer = []
for i in input().split():
    weights.append(int(i))

c = sum(weights)//4 - weights[0] - weights[-1]

answer.append(c)
answer.append(weights[8] - c)
answer.append(weights[9]-(weights[8] - c))
answer.append(weights[1] - c)
answer.append(weights[0]-(weights[1] - c))
answer.sort()

print(*answer)